simulate_agent_paths/AgentLocationSimulator.java

```diff
@@ -0,0 +1,77 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Sends post requests to "https://fos-tracker-278709.an.r.appspot.com/update_agent_location"
+ * for updating latitude and longitude of agent in database whose coordinates are
+ * given in the CoordinatesFile.
+ */
+class ThreadforSendingLocationofAgent extends Thread{
+
+    String CoordinatesFile;
+    ThreadforSendingLocationofAgent(String CoordinatesFile){
+        this.CoordinatesFile = CoordinatesFile;
+    }
+
+    @Override
+    public void run(){
+        try{
+            String email, latitude, longitude;
+            BufferedReader br = new BufferedReader(new FileReader(CoordinatesFile));
+            email = br.readLine();
+            String latitude_longitude = br.readLine();
+            while (latitude_longitude!=null){
+
+                String[] latitude_longitude_tuple = latitude_longitude.split(" ");
+                latitude = latitude_longitude_tuple[0];
+                longitude = latitude_longitude_tuple[1];
+                String jsonRequest = "{\"email\":\""+email+"\",\"latitude\":"+latitude+",\"longitude\":"+longitude+"}";
```

Comment:
Can we use the structs that we defined earlier for this queries to the db? 
It will be better if we can do that as
1. Its easier to make mistakes with strings
2. it more readable

Reply:
I created a toJson() function specific to AgentLocationInfo class. Since this part is not a maven project, I could not add gson dependency. Please review.

Reply:
ok, we can keep this.